src/features/linodes/LinodesDetail/LinodeNetworking/CreateIPv4Drawer.tsx

```
@@ -57,7 +57,7 @@ class CreateIPv4Drawer extends React.Component<CombinedProps, State> {
       .catch(errResponse => {
         this.setState(
           {
-            errors: path(['response', 'data', 'errors'], errResponse)
+            errors: getAPIErrorOrDefault(errResponse)
```

I did this a lot; if I understand correctly, using `path` here, the existing behavior, would swallow a non-API error (like a blocked request), so I feel comfortable replacing this with getAPIErrorOrDefault even though it's a behavior change.